sdk/core/Azure.Core.Experimental/src/Amqp/AmqpMessageBody.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Amqp
+{
+    /// <summary>
+    /// Represents an AMQP message body.
+    /// http://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#section-message-format
```

Comment:
nit:  I'd recommend moving this to `<seealso href="..." />` for consistency with other docs using external references.